pkg/storaged/crypto-tab.jsx

```diff
@@ -38,11 +40,27 @@ import { CryptoKeyslots } from "./crypto-keyslots.jsx";
 
 const _ = cockpit.gettext;
 
+function parse_tag_mtime(tag) {
+    if (tag && tag.indexOf("1:") == 0) {
+        try {
+            const parts = tag.split("-")[1].split(".");
+            const mtime = parseInt(parts[0]) + parseInt(parts[1]) * 1e-9;
```

Comment:
> Are you sure that 1e-9 (nanoseconds) is right? Most timestamps in the OS are ms or us.

Yes, pretty sure.  But it would indeed be best to just omit fractional seconds.

Reply:
> This watch tag is documented as an opaque internal value, so I'm not sure how much we can rely on this (especially across architectures, different file systems, and OSes)

I was debating this a lot with myself... we do control it and we could simply document it. Or better yet, expose this information officially in a watch channel somehow.

The alternative is to spawn a separate `stat` call, but that seemed wasteful since we actually have the information we need, just not officially...

Reply:
Right -- I'd just feel a lot better about that if we test this, so that the test makes sure the displayed time stamp is "close to now", instead of "9283213 years from now"

Reply:
I was going to write a test, and noticed that this is actually wrong: The storage page says "Last modified: Today at 8:45 AM", while the server system time is 6:45 AM. So that moment() thing failed to take into account the time zone. I'll fix this and re-push once the current round of tests is done. Then I'll add an integration test.

Reply:
I checked this again, and indeed our "Server time:" is a bit special -- it's exactly the place which is *not* browser time. So let's keep this as it is, but we somehow need to get the browser's current hour for the integration test.

Reply:
Integration test added, resolving.